src/Sulu/Bundle/MediaBundle/DependencyInjection/Configuration.php

```diff
@@ -204,6 +205,38 @@ private function addStorageSection(ArrayNodeDefinition $node)
                 ->end();
         }
 
+        if (class_exists(AwsS3Adapter::class)) {
+            $storages[] = 'aws_s3';
+            $storagesNode
+                ->arrayNode('aws_s3')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('key')->isRequired()->end()
+                        ->scalarNode('secret')->isRequired()->end()
+                        ->scalarNode('region')->isRequired()->end()
+                        ->scalarNode('bucket_name')->isRequired()->end()
+                        ->scalarNode('version')->defaultValue('latest')->end()
+                        ->scalarNode('endpoint')->defaultValue('latest')->end()
+                        ->scalarNode('segments')->defaultValue(10)->end()
+                    ->end()
+                ->end();
+
+            $storages[] = 'minio';
+            $storagesNode
+                ->arrayNode('minio')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('key')->isRequired()->end()
+                        ->scalarNode('secret')->isRequired()->end()
+                        ->scalarNode('endpoint')->isRequired()->end()
+                        ->scalarNode('bucket_name')->isRequired()->end()
+                        ->scalarNode('region')->defaultValue('us-east-1')->end()
```

Comment:
why here a default?
and would be better if the order of the parameters are the same as above

Reply:
this is the default value in minio configuration but it can be set individually.